src/Lecture1.hs

```diff
@@ -43,27 +44,32 @@ their squares.
 >>> sumOfSquares 3 4
 25
 
+
 >>> sumOfSquares (-2) 7
 53
 
+
 Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+sumOfSquares:: Int -> Int -> Int
+sumOfSquares x y = x*x + y*y
 
 {- | Implement a function that returns the last digit of a given number.
 
 >>> lastDigit 42
 2
 >>> lastDigit (-17)
-7
+WAS WAS 7
+WAS NOW 3
+NOW 7
 
 🕯 HINT: use the @mod@ function
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit:: Int -> Int
+lastDigit n = mod (abs n) 10
```

Comment:
Absolutely correct 👍🏻 
Alternatively, you can use the infix form of a function by putting it inside backticks :slightly_smiling_face: 
```suggestion
lastDigit n = abs n `mod` 10
```

Reply:
Thansk for the feedback!